### ISSUE TITLE
Need Help with Usage History for LaborWorks Account # 120-AR8445

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#932]
In GitLab by @kdjstudios on Jun 6, 2018, 16:33

**Submitted by:** "Lisa Gray" <lisa.gray@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-06-06-58532/conversation

**Server:** Internal

**Client/Site:** 120

**Account:** AR8445

**Issue:** 

The client is requesting usage history for April and May 2018 and the program is showing that it stops before the end of April.  Can you please help me out with this?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 6, 2018, 16:40

@smasih @pchaudhary @tim.traylor 

I am at a loss on what is going on with this account. 

- I pulled the usage history: [AR8445_UsageHistory.pdf](/uploads/ae3574a3bd7e764c544ee2451a72fce9/AR8445_UsageHistory.pdf) and from what I can understand there is over 50+ line items for "VCC Talk Minutes 55062"; "VCC Patch Count 55050"; "120 Operator Time 0501" and "120 Patch Calls 0506" on the 4/28 billing cycle. How is this even possible?
- I looked at the invoice for that billing cycle: [invoice_120-15460.pdf](/uploads/72b083382a6cefbe106299017033c083/invoice_120-15460.pdf) and from what I can understand there was only one item that was usage based??
- The HR report is not working?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 6, 2018, 16:40

I wrote:

> Thank you for notifying us. We have informed the support team and will follow up accordingly.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jun 7, 2018, 05:15

Hi [Kyle] (@kdjstudios ),  

We spent the time to find the cause of the issue, but unable to resolve.
Below are our findings of the investigation:  

*  We pulled the [west_memphis.csv](/uploads/f9a41bd90de8f46640b906a972758a1c/wm.csv) from the server & not found any **resource id** with the same as **account's resource id**. We're still not sure that how's that transactions are in the invoice itself.

*  We still have to look further in this.

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 8, 2018, 08:23

@smasih @pchaudhary @tim.traylor 

Gary wrote:

> Can't you simply use the human readable report?

I responded:

> The support team is currently looking into this as it appears there is an issue with the account. Currently the HR report is not loading and the Usage report has variations that do not seem correct also. Once the support team has an update we will post it.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 8, 2018, 10:38

Lisa Wrote:

> There is nothing with the new resource code on the billing now, the error reads:
> 
> 
>  
> 
> Sorry..!! This Account With Resource ID: 1208445 Was Not Found Within Usage Of This Billing Cycle May 01, 2018
> 
> We merged the Memphis accounts with West Memphis accounts in March Gary, and we have had double-billing on about a dozen plus Memphis accounts, missing reports, blank reports, all sorts of problems and nothing is being done for us.  We had no missing accounts before and our exception reports were a few pages, now I have 60+ missing accounts every month and hundreds of pages of exception reports.  We have been going through them to fix them, make sure all resource codes match, but it is not fixing anything.
> 
> 
>  
> 
> Can you please help me Gary?

@tim.traylor 

I do not recall having the support team merge these two site's accounts? This would appear to possibly be due to operational user error during the merge and not checking the accounts; I would presume then?

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jun 12, 2018, 05:58

Hi [Kyle] ( @kdjstudios ) & [Tim] ( @tim.traylor ),   

Agreed, even we also don't recall the transfer of the Memphis accounts. Let's bring it on today's call & discuss it with Tim before we schedule it for Thursday's call.

Thanks

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 12, 2018, 10:17

The Aavaz team will check a few of the usage files on the server and find out how the account is having usage associated it with.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jun 14, 2018, 06:05

@kdjstudios @tim.traylor 

In the old data on staging server we found that this account [+AR8445 LaborWorks Balance+]  has **resource ID `8445`** which is obviously different than one we have now **`1208445`**.

So, as far as we can understand ... up till april the account had `8445` as a resource id which later got changed, due to which we were not able to find any usage in the [+april usage file+] with new resource id **but found multiple entries under `8445`**
[120_april_30_complete_usage_error.csv](/uploads/121d2cdd9d28674ecb70583d409149c4/120_april_30_complete_usage_error.csv)

This is also the reason why HR report is also not working because, it's looking for `1208445` instead of `8445`.

Bottom line, there is no system generated error.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 14, 2018, 08:50

@smasih @tim.traylor 

Please discuss with @tim.traylor today. 

This is due to when the two sites merged (Memphis and West Memphis) they were to prepend their site number number to the account numbers in order to prevent duplicate accounts. It will need to be discussed what is needed to be done to insure historical data is still view able.

Also, I do not see how this account is showing data on the usage report when you state the account number does not have any? This to me seems like a major system error.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jun 15, 2018, 24:49

@kdjstudios 

> This is due to when the two sites merged (Memphis and West Memphis) they were to prepend their site number number to the account numbers in order to prevent duplicate accounts.

agreed but, you also know the importance of resource_id, it uses this value to associate usage with the respective account.



> Also, I do not see how this account is showing data on the usage report when you state the account number does not have any? This to me seems like a major system error.

Resource Id was changed probably after april 30 before that resource id for this account was `8445`

and if you see in the usage csv there are multiple entries under the same resource id.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 15, 2018, 13:02

@smasih 

Thank you for the clarification. I however do not see a solution proposed on how to resolve this issue? Marking for discussion.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 19, 2018, 10:02

Discussed during today's call and @tim.traylor  and I will need to further discuss this and communicate with Operations to insure accounts are setup correctly in Motion and OA; once that is setup correctly, we will then further discuss what is needed to setup/merge/update the accounts in SAB.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 28, 2018, 14:22

I sent @tim.traylor an email on the 19th with an outline of the process to resolve the merge between the two sites accordingly. I will be reaching out to @tim.traylor to further review this and then send to Operations to start this process.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 29, 2019, 13:51

@amishra @smasih @pchaudhary 

I believe the merge between Memphis and West Memphis was completed successfully and we should be able to close this ticket, correct?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Feb 11, 2019, 13:15

I wrote:

> We are just following up to confirm we may close this ticket. We believe this was resolved with the overhaul of the merging of accounts with Memphis and West Memphis that occurred in September 2018. Please let us know if everything is working as expected or if you are still encountering any issues.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Feb 12, 2019, 10:07

closed
